packages/account-export/src/index.ts

```diff
@@ -1,4 +1,10 @@
-export { setupExportSelectorModal } from "./lib";
+export {
+  setupExportSelectorModal,
+  encodeAccountData,
+  decodeAccountData,
+  encryptAccountData,
```

Comment:
Is there a good case for exporting the `encryptAccountData` method if it's being used internally by the modal workflow? My only concern is that developers might think they need to use that method when it's actually something we're already handling.

Reply:
Totally agree, I will remove them from export